pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[tool.poetry]
+name = "oipie-api-core"
+version = "0.0.1"
+description = ""
+authors = ["Your Name <you@example.com>"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+dependency-injector = "4.40.0"
+Flask = "2.2.2"
+Flask-Migrate = "3.1.0"
+Flask-SQLAlchemy = "2.5.1"
+gunicorn = "20.1.0"
+psycopg2-binary = "2.9.3"
```

Comment:
I think we only need `psycopg2-binary` because `SQLAlchemy` needs it

Reply:
That package is some kind of "peer" dependency, because SQLAlchemy does not install it.